src/components/LocalePicker.tsx

```diff
@@ -42,7 +47,10 @@ function LocalePicker({preferredLocale = CONST.LOCALES.DEFAULT, size = 'normal'}
 
                 App.setLocale(locale);
             }}
+            isDisabled={shouldDisablePicker}
             items={localesToLanguages}
+            shouldAllowDisabledStyle={false}
+            shouldShowOnlyTextWhenDisabled={false}
```

Comment:
I'm not quite understanding why these two props are necessary. Can they just be derived from the `isDisabled` prop?